### ISSUE TITLE
Importing error

### ISSUE THREAD

**Issue by iansedano** [fabiorino/crelly-slider#67]
I am maintaining someone else's wordpress site and when I try to import a slider I get this error from the console

JQMIGRATE: Migrate is installed, version 1.4.1
VM1248:1 Uncaught SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at Object.success (admin.js?ver=1.3.6:1810)
    at i (jquery.js?ver=1.12.4-wp:2)
    at Object.fireWith [as resolveWith] (jquery.js?ver=1.12.4-wp:2)
    at x (jquery.js?ver=1.12.4-wp:4)
    at XMLHttpRequest.c (jquery.js?ver=1.12.4-wp:4)

and the main admin page says "please wait" but nothing happens.
I am attaching the zip file (it is the exact same zip file I exported from crelly)

[crellyslider-selected_works.zip](https://github.com/fabiorino/crelly-slider/files/4522336/crellyslider-selected_works.zip)

If I can provide anything else please let me know!

**Comment by fabiorino** [comment]
Hi, that's weird, it looks like it triggers the success function, but clearly something goes wrong. Are you able to help me debug this? Unfortunately it requires some code modification to print the error message.

Edit the file wp-content/plugins/crelly-slider/wordpress/js/admin.js, add `console.log(response);` before line 1810 and after. Then try importing the slider and see what it prints in the browser console.

**Comment by iansedano** [comment]
![image](https://user-images.githubusercontent.com/38072365/82021885-fb5cb980-968b-11ea-8a0f-26d86bdd89e1.png)

I added in the lines you mentioned, but still getting the same error message - nothing new in the console.

![image](https://user-images.githubusercontent.com/38072365/82021974-20512c80-968c-11ea-8761-4b555b38f1de.png)

and the wordpress admin page:

![image](https://user-images.githubusercontent.com/38072365/82022015-33fc9300-968c-11ea-876f-e8194cb72805.png)

**Comment by fabiorino** [comment]
Hi, another couple of things to check: do the images get uploaded? You should see them in the WordPress media gallery. Does the slider show up in the list of existing sliders?

**Comment by iansedano** [comment]
Nope, no uploaded images and the slider does not show up. Its as if nothing happened.
